package/MDAnalysis/analysis/dssp/dssp.py

```diff
@@ -11,58 +11,69 @@
 The module contains code to build hydrogend bond contact map,
 and use it to assign protein secondary structure (:class:`DSSP`).
 
-This module uses the python version of the original algorithm by Kabsch & Sander (1983),
-re-implemented by @ShintaroMinami and available under MIT license
-[here](https://github.com/ShintaroMinami/PyDSSP/tree/master#differences-from-the-original-dssp)
-Note that this implementation does not discriminate different types of beta-sheets,
-as well as different types of helices, meaning you will get 3_10 helices
-and pi-helices labelled as "helix" too.
+This module uses the python version of the original algorithm :footcite:p:`Kabsch1983`,
+re-implemented by @ShintaroMinami and available under MIT license from
+`ShintaroMinami/PyDSSP <https://github.com/ShintaroMinami/PyDSSP/tree/master#differences-from-the-original-dssp>`_.
+
+Note that this implementation does not discriminate different types of
+beta-sheets, as well as different types of helices, meaning you will get
+:math:`3_{10}` helices and π-helices labelled as "helix" too.
 
 
 Using original `pydssp`
-----------------------
+-----------------------
+
+The default implementation uses the original `pydssp` (v.0.9.0) code, rewritten
+without usage of `einops` library and hence having no dependencies. If you want
+to explicitly use `pydssp` (or its particular version), install it to your
+current environment with `python -m pip install pydssp`. Please note that the
+way MDAnalysis uses pydssp does not support pydssp's capability for batch
+processing or its use of the pytorch library.
```

Comment:
reST formatting `.neq.` markdown :-)
```suggestion
The default implementation uses the original *pydssp* (v.0.9.0) code, rewritten
without usage of the *einops* library and hence having no dependencies. If you want
to explicitly use *pydssp* (or its particular version), install it to your
current environment with ``python -m pip install pydssp``. Please note that the
way MDAnalysis uses *pydssp* does not support *pydssp* 's capability for batch
processing or its use of the *pytorch* library.
```

I just use explicit emphasis (italics) fir package names. I am not 100% sure that we have a style guide entry for how to format them. Mono-spaced is two back-ticks.